README.md

```diff
@@ -70,10 +82,11 @@ docker save "crossplane/stack-minimal-gcp:<version>" | (eval "$(minikube docker-
 
 After running this, you can use the [installation](#installation) command and the image loaded into minikube node will be picked up. 
 
-[network]: https://github.com/crossplane/crossplane/blob/master/docs/api/crossplaneio/stack-gcp/compute-gcp-crossplane-io-v1beta1.md#network
-[subnetwork]: https://github.com/crossplane/crossplane/blob/master/docs/api/crossplaneio/stack-gcp/compute-gcp-crossplane-io-v1beta1.md#subnetwork
-[global-address]: https://github.com/crossplane/crossplane/blob/master/docs/api/crossplaneio/stack-gcp/compute-gcp-crossplane-io-v1beta1.md#globaladdress
-[connection]: https://cloud.google.com/service-infrastructure/docs/service-networking/reference/rest/v1/services.connections
-[gkecluster-class]: https://github.com/crossplane/crossplane/blob/master/docs/api/crossplaneio/stack-gcp/compute-gcp-crossplane-io-v1alpha3.md#gkeclusterclass
-[cloudmemorystoreinstance-class]: https://github.com/crossplane/crossplane/blob/master/docs/api/crossplaneio/stack-gcp/cache-gcp-crossplane-io-v1beta1.md#cloudmemorystoreinstanceclass
-[cloudsqlinstance-class]: https://github.com/crossplane/crossplane/blob/master/docs/api/crossplaneio/stack-gcp/database-gcp-crossplane-io-v1beta1.md#cloudsqlinstanceclass
\ No newline at end of file
+[network]: kustomize/gcp/compute/network.yaml
```

Comment:
It looks like we were linking to `stack-gcp` before, not the minimal stack. Do we still want to be linking to the gcp stack objects somewhere?

Reply:
Yeah, I thought that since the list is titled as what this stack deploys, it'd make more sense for links to carry you to the file that is actually deployed. GCP docs felt irrelevant at that level.